apigentools/commands/generate.py

```diff
@@ -250,3 +261,20 @@ def run(self):
             self.write_dot_apigentools_info(language)
 
         return 0
+
+    def pull_repository(self, language):
+        output_dir = self.get_generated_lang_dir(language.language)
+        if self.args.use_https:
+            repo = REPO_HTTPS_URL.format(language.github_org, language.github_repo)
+        else:
+            repo = REPO_SSH_URL.format(language.github_org, language.github_repo)
+
+        try:
+            # Git doesn't allow you to clone into a non empty dir
+            # so create a temp dir and we'll move the content after
+            with tempfile.TemporaryDirectory() as temp_repo_dir:
+                run_command(['git', 'clone', '--depth=2', repo, temp_repo_dir])
```

Comment:
Why `--depth=2`?

Reply:
Figured a shallow clone was quicker since we don't need the full history here.

Reply:
Allright, fair. We can always change it (or add an argument that would allow users to choose) without breaking backwards compatibility, so let's keep it like this.